docs/guide/exe_verilator.rst

```diff
@@ -777,6 +777,16 @@ Summary:
    for deparametrized modules with :option:`/*verilator&32;hier_block*/`
    metacomment. See :ref:`Hierarchical Verilation`.
 
+.. option:: --hierarchical-threads <threads>
+
+   Specifies the number of threads used for scheduling hierarchical blocks.
```

Comment:
Need a bit more information on what this means to the user.  E.g. I can think a user wondering is this the thread that each hierarchical block uses, or sum of all hierarchical blocks?

Reply:
@wsnyder Thanks for resolving this in https://github.com/verilator/verilator/commit/40881d7e79061b0954e0b58747eaea3db821d630